doc/cli/operator-sdk_build.md

```diff
@@ -29,6 +30,7 @@ operator-sdk build <image> [flags]
   -h, --help                      help for build
       --image-build-args string   Extra image build arguments as one string such as "--build-arg https_proxy=$https_proxy"
       --image-builder string      Tool to build OCI images. One of: [docker, podman, buildah] (default "docker")
+      --skip-image                If set as true, the build of the image will be skipped and it will just compile and prepare the Dockerfile.
```

Comment:
```suggestion
      --skip-image                If set, the build of the image will be skipped and it will just compile and prepare the Dockerfile.
```

Since the default value of it is true.